src/index.html

```diff
@@ -7,8 +7,61 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:wght@400;700&family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <label class="link__home nav__item">
+          <a href="#home">
+            <img src="images/logo.png" alt="logo">
+          </a>
+        </label> 
+        <nav class="navigation"> 
```

Comment:
It should be `<nav class="nav">` if you use `nav__list` naming for BEM elements.